packages/server/dataloader/azureDevOpsLoaders.ts

```diff
@@ -507,23 +566,21 @@ export const azureDevOpsWorkItems = (
             console.log(error)
             return []
           }
-          const mappedWorkItems: AzureDevOpsWorkItem[] = returnedWorkItems.map(
-            (returnedWorkItem) => ({
-              id: returnedWorkItem.id.toString(),
-              title: returnedWorkItem.fields['System.Title'],
-              teamProject: getProjectId(new URL(returnedWorkItem.url)),
-              url: returnedWorkItem.url,
-              state: returnedWorkItem.fields['System.State'],
-              type: returnedWorkItem.fields['System.WorkItemType'],
-              descriptionHTML: returnedWorkItem.fields['System.Description']
-                ? returnedWorkItem.fields['System.Description']
-                : '',
-
-              service: 'azureDevOps'
+
+          const mappedWorkItems: AzureDevOpsWorkItem[] = await Promise.all(
+            returnedWorkItems.map(async (returnedWorkItem): Promise<AzureDevOpsWorkItem> => {
+              const mappedWorkItem = await getMappedAzureDevOpsWorkItem(
```

Comment:
+1 rather than awaiting in each loop, can we return an array of promises and then use Promise all?